python/py-bottlenose/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        lionheart bottlenose 1.1.10
```

Comment:
any reason not to take the latest release, or did you meant to type 1.1.18? Actually, that is confusing... PyPI says 1.18p but the latest tag on GitHub is 1.1.10 - can you look into that?

Reply:
also, please specify where to get the tarball from when using the github PG. "releases" is preferred if upstream does that, next-best option is "archive". If you change that you might need to update the checksums as well (don't forget to do port clean first).

Reply:
@reneeotten PyPi says that the last release is `1.1.8`, not `1.1.18` ;)

Reply:
oh well... reading is apparently difficult on the Saturday morning... the point about `github.tarball_from` is still relevant though - correct?

Reply:
@reneeotten upstream doesn't serve releases => as you may see I've added `github.tarball_from archive`.

I expect that `port clean` cleans also distfile but it doesn't => I've re-run it with `--all` and here an updated checksums.